doc/translating.rst

```diff
@@ -0,0 +1,17 @@
+====================================
+Translating the SQUAD user interface
+====================================
+
+Just run `./scripts/translate` with your language code as the only parameter.
+For example, for Brazilian Portuguese::
+
+    $ ./scripts/translate pt_BR
+    squad/core/locale/pt_BR/LC_MESSAGES/django.po: 0 translated messages, 22 untranslated messages.
+    squad/frontend/locale/pt_BR/LC_MESSAGES/django.po: 0 translated messages, 212 untranslated messages.
+
+The script will output the names of the translation files that need to be
+edited, together with the current translation statistics. Edit the `*.po`
+files, commit, and send a pull request.
+
+This procedure is valid both for creating new translations, and for updating
```

Comment:
When running `./scripts/translate`, there's no new messages added to the po files, i.e. if I add a new `_()` or `N_()` to the code. Is that the intention here or should the user run `./scripts/update-translation-files` before that?

Reply:
`./scripts/update-translation-files` should be run by a maintainer when close to a release, and when no more strings should be added before the release (this is usually called "string freeze").

`./scripts/translate` should be run by each translator.

Reply:
didn't know about "string freeze", but grasped it by context :)